.github/workflows/build_image_milvus_arm.yml

```diff
@@ -160,37 +160,34 @@ jobs:
         if: ${{ !startsWith(env.SOURCE_BRANCH,'rel-') && env.SOURCE_BRANCH != 'main' }}
         shell: bash
         run: |
-          chmod +x ./images/bookie/script.sh
-          chmod +x ./images/zookeeper/generate-zookeeper-config.sh
+          sudo docker buildx use multiBuild
           while IFS=$'\t' read -r image function file _; do
             if [[ "$function" == "dockerBuild" && "$file" != "null" ]]; then
               echo "Building $image Image"
-              sudo docker build -f $file --no-cache -t ${{env.DOCKERHUB_ORG}}/$image .
+              sudo docker buildx build -f $file --no-cache -t ${{env.DOCKERHUB_ORG}}/$image --platform=linux/amd64,linux/arm64 .
             fi
           done < <(yq e '.generates.dockerImages[] | [.name, .buildFunction, .dockerfile] | @tsv' convoy.yaml)
 
       - name: Build and push to tessellProd
         if: ${{ (startsWith(env.SOURCE_BRANCH,'rel-') || env.SOURCE_BRANCH == 'main')}}
         shell: bash
         run: |
+          sudo docker buildx use multiBuild
           while IFS=$'\t' read -r image function file _; do
             if [[ "$function" == "dockerBuild" && "$file" != "null" ]]; then
               echo "Building $image Image"
-              sudo docker build -f $file --no-cache -t ${{env.DOCKERHUB_ORG}}/$image:${{env.TAG}} .
-              sudo docker push ${{env.DOCKERHUB_ORG}}/$image:${{env.TAG}}
-              sudo docker tag ${{env.DOCKERHUB_ORG}}/$image:${{env.TAG}} ${{vars.TESSELL_DEV_ECR}}/$image:${{env.TAG}}
-              sudo docker push ${{vars.TESSELL_DEV_ECR}}/$image:${{env.TAG}}
+              sudo docker buildx build --push -f $file --no-cache -t ${{env.DOCKERHUB_ORG}}/$image:${{env.TAG}} --platform=linux/amd64,linux/arm64 .
+              sudo docker build --push -f $file --no-cache -t ${{vars.TESSELL_DEV_ECR}}/$image:${{env.TAG}} .
             fi
           done < <(yq e '.generates.dockerImages[] | [.name, .buildFunction, .dockerfile] | @tsv' convoy.yaml)
 
           while IFS=$'\t' read -r image version _; do
             echo $image
             echo $version
             sudo docker pull ${{env.DOCKERHUB_ORG}}/$image:$version
-            sudo docker tag ${{env.DOCKERHUB_ORG}}/$image:$version ${{env.DOCKERHUB_ORG}}/$image:${{env.TAG}}
-            sudo docker push ${{env.DOCKERHUB_ORG}}/$image:${{env.TAG}}
-            sudo docker tag ${{env.DOCKERHUB_ORG}}/$image:${{env.TAG}} ${{vars.TESSELL_DEV_ECR}}/$image:${{env.TAG}}
+            sudo docker tag ${{env.DOCKERHUB_ORG}}/$image:$version ${{vars.TESSELL_DEV_ECR}}/$image:${{env.TAG}}
             sudo docker push ${{vars.TESSELL_DEV_ECR}}/$image:${{env.TAG}}
+            sudo docker buildx imagetools create -t ${{env.DOCKERHUB_ORG}}/$image:${{env.TAG}} ${{env.DOCKERHUB_ORG}}/$image:$version
```

Comment:
You need to remove sudo (separate fix) 
https://tessell.atlassian.net/browse/TDEVOPS-1428